src/components/CardInput/CardInput.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { MantineProvider } from '@/components/MainProvider'
+
+import '@mantine/core/styles.css'
+import { theme } from '@/components/theme'
+
+import { CardInput } from '@/components/CardInput'
+import { ChangeEvent, useState } from 'react'
+
+// ----------------------------------------------------------------------
+
+const CardInputWrapper = () => {
+  const [value, setValue] = useState('')
+
+  const changeHandler = (e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value)
```

Comment:
```suggestion
  const handleChange = (e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value)
```

Давай обработчики называть как раньше, handle + сущность (если необходимо Input, Select и тд) + событие (change, click и тд)